babel.cfg

```diff
@@ -1,6 +1,6 @@
 [python: **.py]
 [jinja2: **/templates/**.jinja2]
 extensions=jinja2.ext.autoescape,jinja2.ext.with_,jinja2.ext.do
-[javascript: **/templates/js/**.js]
+[javascript-jinja2: **/templates/js/**.js.jinja2]
```

Comment:
These templates don't need to be in a `js` subfolder if they have the appropriate file extension.